Debian/8/input/oval/oval_5.11/templates/Makefile

```diff
@@ -3,10 +3,16 @@ OUTPUT:=$(shell mkdir -p output)
 
 all:
 	${SHARED_DIR}/create_services_enabled.py services_enabled.csv
+	${SHARED_DIR}/create_services_disabled.py services_disabled.csv
 
-copy:
+copy: postbuild
 	cp output/*.xml ..
 
+# some shared oval need postbuild update.
+postbuild:
+#	ssh service is named ssh in Debian, not sshd, updating oval id.
+	sed -re 's:service_ssh_disabled:service_sshd_disabled:g' -i output/service_ssh_disabled.xml
```

Comment:
I'm currently correcting the code based on your remarks. @redhatrises : I've corrected the ssh vs sshd naming. My mistakes came from the current integration of systemd in Debian jessie, keeping sysvinit in the same time for retro-compatibility. One using ssh as service name, the other sshd...
I've updated for systemd compliance, cleaned the sed (and the postbuild).